document/web-api/index.bs

```diff
@@ -2,7 +2,7 @@
 Title: WebAssembly Web API
 Shortname: wasm-web-api
 Group: wasm
-Status: FPWD
+Status: w3c/FPWD
```

Comment:
This change was needed for me to be able to build locally

Reply:
If you update your bikeshed version that should fix it (it will then know wasm is a w3c project).
The reason I changed these is that bikeshed rejects some of the normal stamping with the full string. I still owe Tab filing an issue on that.

Reply:
How should I update it? I tried `bikeshed update` and it still gave this error.

Reply:
I think you need to actually update your Bikeshed source code, so e.g. go to the director you checked out bikeshed and run `git pull`.